tests/js/docker-couchdb-tests/basic-tests.js

```diff
@@ -14,24 +14,38 @@ require("gpii-express");
 gpii.express.loadTestingSupport();
 
 require("../pouch-config");
+require("../lib/caseHolder");
+
+fluid.registerNamespace("gpii.tests.pouch.basic");
+
+
+gpii.tests.pouch.basic.checkRecordAndStartDelete = function (response, body, expectedStatus, expectedBody, deleteRequest) {
+    var record = JSON.parse(body);
+    gpii.test.pouch.checkResponse(response, body, expectedStatus, expectedBody);
+
+    // DELETE requests must reference a specific revision, as in:
+    // DELETE /recipes/FishStew?rev=1-9c65296036141e575d32ba9c034dd3ee
+    deleteRequest.send({}, { termMap: { rev: record._rev } });
+};
 
 fluid.defaults("gpii.tests.pouch.basic.caseHolder", {
```

Comment:
Odd that pouch has been replaced with pouchdb in the base gradeName but not throughout

Reply:
Seems like the waters are pretty muddy.  Almost all of the code in `./src` uses "pouch", but a lot of the tests, including the stuff that people are already using (like the `testEnvironment`) use "pouchdb".

I would rather not break people's existing tests by renaming, but I'm already increasingly uncomfortable with either the "pouch" or "pouchdb" namespace for the docker work.  Maybe it's time to spin off the docker bits into a new micromodule with a name that actually reflects its purpose, like `gpii-couch-test-harness` or something shorter.  If we did that, we could pare down gpii-pouchdb to a smaller module that only supports the pouch-specific use cases we agree are valuable (standalone use in the browser, possibly node, depending on Cindy's input).  IMO this is a good discussion for the meeting tomorrow.